data/sql/updates/pending_db_world/rev_1584295889272307600.sql

```diff
@@ -0,0 +1,209 @@
+-- Harbinger Skyriss
+UPDATE `creature_template` SET `mindmg` = 969, `maxdmg` = 1370, `DamageModifier` = 1 WHERE `entry` = 20912;
+-- ===================================================================================================================
+
+
+-- Tempest Keep: The Botanica (67 - 73)
+-- -------------------------------------------------------------------------------------------------------------------
+-- Commander Sarannis
+UPDATE `creature_template` SET `mindmg` = 1523, `maxdmg` = 2154, `DamageModifier` = 1 WHERE `entry` = 17976;
+-- High Botanist Freywinn
+UPDATE `creature_template` SET `mindmg` = 1413, `maxdmg` = 1997, `DamageModifier` = 1 WHERE `entry` = 17975;
+-- Thorngrin the Tender
+UPDATE `creature_template` SET `mindmg` = 2001, `maxdmg` = 2826, `DamageModifier` = 1 WHERE `entry` = 17978;
+-- Thorngrin the Tender [Heroic]
+UPDATE `creature_template` SET `mindmg` = 3394, `maxdmg` = 4796, `DamageModifier` = 1 WHERE `entry` = 21581;
+-- Laj
+UPDATE `creature_template` SET `mindmg` = 1960, `maxdmg` = 2772, `DamageModifier` = 1 WHERE `entry` = 17980;
+UPDATE `creature_template` SET `resistance5` = 500 WHERE `entry` = 17980;
+-- ===================================================================================================================
+
+-- Tempest Keep: The Mechanar (67 - 73)
+-- -------------------------------------------------------------------------------------------------------------------
+-- Mechano-Lord Capacitus
+UPDATE `creature_template` SET `mindmg` = 2091, `maxdmg` = 2957, `DamageModifier` = 1 WHERE `entry` = 19219;
+-- Gatewatcher Gyro-Kill
+UPDATE `creature_template` SET `mindmg` = 2091, `maxdmg` = 2957, `DamageModifier` = 1 WHERE `entry` = 19218;
+-- Gatewatcher Iron-Hand
+UPDATE `creature_template` SET `mindmg` = 2091, `maxdmg` = 2957, `DamageModifier` = 1 WHERE `entry` = 19710;
+-- Nethermancer Sepethrea
+UPDATE `creature_template` SET `mindmg` = 1576, `maxdmg` = 2227, `DamageModifier` = 1 WHERE `entry` = 19221;
+-- Pathaleon the Calculator
+UPDATE `creature_template` SET `mindmg` = 848, `maxdmg` = 1198, `DamageModifier` = 1 WHERE `entry` = 19220;
+-- ===================================================================================================================
+
+-- The Black Morass (68 - 73)
+-- -------------------------------------------------------------------------------------------------------------------
+-- Chrono Lord Deja
+UPDATE `creature_template` SET `mindmg` = 1455, `maxdmg` = 2056, `DamageModifier` = 1 WHERE `entry` = 17879;
+-- Temporus
+UPDATE `creature_template` SET `mindmg` = 1829, `maxdmg` = 2587, `DamageModifier` = 1 WHERE `entry` = 17880;
+-- Aeonus
+UPDATE `creature_template` SET `mindmg` = 2353, `maxdmg` = 3327, `DamageModifier` = 1 WHERE `entry` = 17881;
+-- ===================================================================================================================
```

Comment:
Add a newline to the end of the file
```suggestion
-- ===================================================================================================================

```